tests/unit/test_utils.py

```diff
@@ -38,7 +38,7 @@
 )
 from pip._internal.utils.setuptools_build import make_setuptools_shim_args
 from pip._internal.utils.temp_dir import AdjacentTempDirectory, TempDirectory
-from pip._internal.utils.ui import SpinnerInterface
+from pip._internal.cli.spinners import SpinnerInterface
```

Comment:
Tests for this should probably move into a `test_cli_spinners.py` file. :)

Reply:
Sure will do that.